.github/workflows/install_deps/action.yml

```diff
@@ -98,7 +98,8 @@ runs:
     - name: Install Catalyst
       shell: bash
       if: inputs.install_catalyst_after_pennylane == 'true'
-      run: pip install --upgrade pennylane-catalyst
+      # TODO: replace after release
+      run: pip install --extra-index-url https://test.pypi.org/simple/ pennylane-catalyst --pre
```

Comment:
TODO: add version after https://github.com/PennyLaneAI/pennylane/pull/6117 is merged and wheels are on test pypi